tests/test_config_parsing/test_res_config.py

```diff
@@ -91,3 +94,26 @@ def test_res_config_parses_date():
     expected_ens_path = f"{expected_storage}/ensemble"
     assert res_config.model_config.getEnspath() == expected_ens_path
     assert res_config.model_config.getRunpathAsString() == expected_run_path
+
+
+@pytest.mark.usefixtures("use_tmpdir")
+@given(config_dicts())
+def test_res_config_throws_on_missing_forward_model_job(config_dict):
+    cwd = os.getcwd()
+    filename = config_dict[pytest.TEST_CONFIG_FILE_KEY]
```

Comment:
depending on the discussion later today i guess the lines like this is not valid anymore?

Reply:
sure! i can fixup after the meeting, with whatever we arrive at, or perhaps do it in a separate PR, if Eivind's idea would like to live in its own PR : )

Reply:
it's gone! not needed in these tests anyways, so whatever we do elsewhere, it's removed here, and that is the right thing to do here.